entries/s/sendowl.com.json

```
@@ -0,0 +1,12 @@
+{
+    "SendOwl": {
+      "domain": "sendowl.com",
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://help.sendowl.com/help/two-factor-authentication",
+      "keywords": [
+        "other"
```

Is it similar to Stripe? Because Stripe is in the Payments category.
```suggestion
        "payments"
```

---

No, it's not like Stripe. Stripe is payment processor. SendOwl is about product selling flow.
SendOwl is more like ThriveCart which is in "other" category.